pkg/test/resourcefixture/testdata/basic/essentialcontacts/v1alpha1/essentialcontactscontact/essentialcontactscontact-minimal/_http.log

```diff
@@ -0,0 +1,145 @@
+POST https://essentialcontacts.googleapis.com/v1/projects/${projectId}/contacts?%24alt=json%3Benum-encoding%3Dint
```

Comment:
Is the test log generated against mock? Can we also test against real (which ensures the controller actually works) and see the diff in test log?

Reply:
done